scrabbleScoreExercise/scrabbleScore.js

```diff
@@ -0,0 +1,69 @@
+// Programme: takes each letter of string input and saves it in an array
+//loops through each letter, saves corresponding score for the letter
+// scores are then summed up and scrabble score is displayed back to user
+'use strict'
+
+const score1 = ['a', 'e','i','o','u', 'l','n','r','s','t'];
+const score2 = ['d','g'];
+const score3 = ['b', 'c','m','p'];
+const score4 = ['f','h','v','w','y'];
+const score5 = ['k'];
+const score8 = ['j','x'];
+const score10 = ['q','z'];
+
+let scrabbleWord = prompt('Enter your scrabble word: ').toLowerCase();
+console.log(scrabbleWord);
+
+const scrabbleLetters = [];
+const scrabbleLetterScores = [];
+
+for (let i = 0; i<scrabbleWord.length; i++){
+    const letter = scrabbleWord[i]
+    scrabbleLetters.push(letter);
+}
+
+for (let i = 0; i<scrabbleLetters.length; i++) {
+    if (score1.includes(scrabbleLetters[i])){
+        const numberScore = 1
+        scrabbleLetterScores.push(numberScore);
```

Comment:
You could also directly increment `scrabbleWordSum` in each of these places without first pushing to the `scrabbleLetterScores` list

Reply:
Oooh! that makes a lot more sense